src/utils/puppeteer.ts

```diff
@@ -0,0 +1,67 @@
+import { exec } from 'child_process'
+import { promisify } from 'util'
+import os from 'os'
+import path from 'path'
+import { LaunchOptions } from 'puppeteer-core'
+import * as chromeFinder from 'chrome-launcher/dist/chrome-finder'
+
+const execPromise = promisify(exec)
+
+interface PuppeteerLaunchArgs {
+  profile?: string
+}
+
+let executablePath: string | undefined | false = false
+let isWsl: boolean | undefined
+
+export function isWSL(): boolean {
+  if (isWsl === undefined) isWsl = require('is-wsl') as boolean
+  return isWsl
+}
+
+export const resolveWSLPath = async (path: string): Promise<string> =>
+  (await execPromise(`wslpath -m ${path.replace(/\\/g, '\\\\')}`)).stdout.trim()
+
+export function generatePuppeteerLaunchArgs({
+  profile,
+}: PuppeteerLaunchArgs = {}): Partial<LaunchOptions> {
+  // Puppeteer >= v1.13.0 doesn't use BGPT due to crbug.com/937609.
+  // https://github.com/GoogleChrome/puppeteer/blob/master/lib/Launcher.js
+  //
+  // But it causes invalid rendering in Marp's `inlineSVG` mode. So we override
+  // `--disable-features` option to prevent disabling BGPT.
+  const args = new Set<string>(['--disable-features=TranslateUI'])
```

Comment:
The default arguments by Puppeteer is defined `--disable-features=BlinkGenPropertyTrees`, but rendering of Marp's inline SVG mode requires this feature to prevent glitch. Now we've enabled BGPT forcely because PDF conversion on WSL may return empty slides.